index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Kossi Fioklou Portfolio</title>
+</head>
+<body>
+    <!--This is the section for the header-->
+      <nav id="nav-bar">
+        <div id="logo">
+            <a href="#">
+                <img src="My_Logo.png" alt="logo">
+            </a> 
+        </div>
+          <div class="nav-buttons">
+            <a href="#works" class="nav-button">Portfolio</a>
+            <a href="#about" class="nav-button">About</a>
+            <a href="#contact" class="nav-button">Contact</a>
+          </div>
```

Comment:
- [ ] For the mobile version of the portfolio, it is only required that you create a menu icon, and wrap it in a `nav` tag. The menu icon should be a link as well
_Hint - fontawesome icons_